flamegraph.pl

```diff
@@ -1256,12 +1256,18 @@ sub flow {
 $im->filledRectangle(0, 0, $imagewidth, $imageheight, 'url(#background)');
 $im->stringTTF("title", int($imagewidth / 2), $fontsize * 2, $titletext);
 $im->stringTTF("subtitle", int($imagewidth / 2), $fontsize * 4, $subtitletext) if $subtitletext ne "";
-$im->stringTTF("details", $xpad, $imageheight - ($ypad2 / 2), " ");
 $im->stringTTF("dependencies", $xpad, $imageheight - ($ypad2 / 2) + 20, " ");
-$im->stringTTF("unzoom", $xpad, $fontsize * 2, "Reset Zoom", 'class="hide"');
 $im->stringTTF("search", $imagewidth - $xpad - 170, $fontsize * 2, "Search");
 $im->stringTTF("ignorecase", $imagewidth - $xpad - 75, $fontsize * 2, "Ignore-Case");
-$im->stringTTF("matched", $imagewidth - $xpad - 170, $imageheight - ($ypad2 / 2), " ");
+unless ($inverted) {
```

Comment:
Use regular `if`, it is clearer when having an `else` clause

Reply:
The entire codebase uses it like this (even with else clause)- just following conventions

Reply:
OK